### PR TITLE
Add Utils package

### PR DESCRIPTION
**Pull request by motokotomax**

### CODE REVIEW

**Comment by ValentinaIancu-Gini** [comment]
@motokotomax I have one question...when we need to update this Uitls package, we will need to version it, right? Is not adding complexity to what we already have? Or can somehow skip the versioning part of that specific package?

**Comment by motokotomax** [comment]
> @motokotomax I have one question...when we need to update this Uitls package, we will need to version it, right? Is not adding complexity to what we already have? Or can somehow skip the versioning part of that specific package?

I think we can use just main branch (without version). but seems to code in utils package is "pretty constant"
Screenshot 2024-07-29 at 10 51 59

File: GiniComponents/GiniUtilites/Sources/GiniUtilites/Utilites.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+/**
+ Returns a decimal value
+
+ - parameter inputFieldString: String from input field.
+
+ - returns: decimal value in current locale.
+ */
+
+public func decimal(from inputFieldString: String) -> Decimal? {
```
Comment (razvancapra) [inline GiniComponents/GiniUtilites/Sources/GiniUtilites/Utilites.swift:17]: Suggestion: Maybe we should add this to a String extension into our SDK?!

**Review by razvancapra — APPROVED** [review]
Nice job, Max! 👍

File: GiniComponents/GiniUtilites/Sources/GiniUtilites/Utilites.swift (hunk 1 above)
Comment (motokotomax) [reply GiniComponents/GiniUtilites/Sources/GiniUtilites/Utilites.swift]: done

**Pull request merged**